### ISSUE TITLE
Duplicate slides lose uploaded media

### ISSUE THREAD

**Issue by eerotal** [eerotal/LibreSignage#33]
Slide media is not duplicated when a slide is duplicated, which results in broken media urls in slides.

This issue was separated from #27.

**Comment by eerotal** [comment]
This should now be fixed in `next`.

**Comment by infomaniac50** [comment]
Possible regression. 

I tried to copy a slide containing 1 video asset.

The result was a dialog with the text:

```
Internal server error
The server encountered an internal server error. (copy(): Filename cannot be empty)
```

The API in DevTools gave this response.

```json
{
	"error": 1,
	"thrown_at": "\/var\/www\/html\/common\/php\/slide\/slideasset.php @ ln: 170",
	"e_msg": "copy(): Filename cannot be empty",
	"e_trace": "#0 [internal function]: {closure}(2, 'copy(): Filenam...', '\/var\/www\/html\/c...', 170, Array)\n#1 \/var\/www\/html\/common\/php\/slide\/slideasset.php(170): copy('', '\/var\/www\/html\/d...')\n#2 \/var\/www\/html\/common\/php\/slide\/slide.php(164): SlideAsset->clone(Object(Slide))\n#3 \/var\/www\/html\/api\/endpoint\/slide\/slide_dup.php(44): Slide->dup()\n#4 {main}"
}
```

**Comment by eerotal** [comment]
The error occured because the SlideAsset::clone() function didn't check whether the asset thumbnail existed before trying to copy it over to the new slide. Whoops! This should now be fixed in fff622cf558d085a9011227f64c78dfc68c1eea5.
